.editorconfig

```diff
@@ -1,4 +1,4 @@
-# Editor configuration, see https://editorconfig.org
+# Editor configuration, see http://editorconfig.org
```

Comment:
Is this needed?

Reply:
I actually did not do this changes, but just automatically created on `ng new <project>` using **@angular/cli version 4**